streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/TaskMovement.java

```diff
@@ -64,12 +66,10 @@ private static boolean taskIsCaughtUpOnClientOrNoCaughtUpClientsExist(final Task
         return caughtUpClients == null || caughtUpClients.contains(client);
     }
 
-    /**
-     * @return whether any warmup replicas were assigned
-     */
-    static boolean assignTaskMovements(final Map<TaskId, SortedSet<UUID>> tasksToCaughtUpClients,
-                                       final Map<UUID, ClientState> clientStates,
-                                       final int maxWarmupReplicas) {
+    static int assignActiveTaskMovements(final Map<TaskId, SortedSet<UUID>> tasksToCaughtUpClients,
```

Comment:
I changed this to return an int just because it made stepping through the assignment in the debugger a bit easier to understand. It serves no algorithmic purpose.